.yarnrc.yml

```diff
@@ -1 +1,3 @@
 nodeLinker: node-modules
+
+yarnPath: .yarn/releases/yarn-4.5.1.cjs
```

Comment:
**issue:** Are you sure we want to use a local binary path? Nowadays with nvm and corepack this seems rather pointless to me 🤔